src/NServiceBus.TransportTests/ConfigureMsmqTransportInfrastructure.cs

```diff
@@ -10,11 +10,15 @@
 
 class ConfigureMsmqTransportInfrastructure : IConfigureTransportInfrastructure
 {
-    public TransportInfrastructure Configure(SettingsHolder settings)
+    public TransportConfigurationResult Configure(SettingsHolder settings, TransportTransactionMode transactionMode)
```

Comment:
it's there because we have at least ASB that provides implementation of `IConfigureTransportInfrastructure` that needs it `TransportTransactionMode`